database/database.go

```diff
@@ -0,0 +1,23 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func Connect(connection string) (*sql.DB, error) {
+	db, err := sql.Open("postgres", connection)
+
+	if err != nil {
+		panic(err)
```

Comment:
`return nil, err`

Never use panic in package, it can cause unpredictable behaviour.

Reply:
I took it out.